README.md

```diff
@@ -0,0 +1,61 @@
+# Dev environemnt setup
+
+## Terms 
+ - Windows shell instances: (powershell/cmd) doesn't really matter which
+ - WSL: a feature of Windows that allows developers to run a Linux environment without the need for a separate virtual machine
+ - Fedora Remix: a linux distribution
+ - bash: the default shell fedora uses (equivalent to cmd/powershell on windows)
+
+## Prequisites:
+ - What a terminal is
+ - Directorys, Paths, Files
```

Comment:
Typo: Directories

Also, how exactly are these prerequisites?
(What does that mean?)